### PR TITLE
Listing Page: Body-Grid-Layout

### PR DESCRIPTION
**Pull request by jeffworley**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title></title>
+        <title>Etsy Listing Page</title>
```
Comment (al-the-x) [inline src/index.html:6]: Please move this code into `product.html`... Maybe coordinate with @jeremyrist on renaming that to `listing.html` as well.

File: src/index.html [hunk 2]
```diff
@@ -19,10 +19,36 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
-
-        <main class="container">
-        <!-- Add your site or application content here -->
-        </main><!-- .container -->
```
Comment (al-the-x) [inline src/index.html:25]: Better to _link_ to `product.html` / `listing.html` (see above) in here than put all the code for a listing in `index.html`

File: src/scss/_grid.scss [hunk 3]
```diff
@@ -1,26 +1,95 @@
...
+//       }
+//
+//       @if ( $span < $units ){
+//         .pad-#{$span}-#{$units} {
+//           margin-left: 100% * ($span/$units);
+//         }
+//       }
+//     } // END @for $span
+//   } // END @each $units
+// }
```
Comment (al-the-x) [inline src/scss/_grid.scss:26]: Y U NO LIKE MY MAGIC SAUCE!?!

File: src/scss/_grid.scss [hunk 4]
```diff
@@ -1,26 +1,95 @@
-.grid {
-  @extend .clearfix;
+// .continaer {
```
Comment (al-the-x) [inline src/scss/_grid.scss:1]: Maybe that should have been `.container` instead... :disappointed:

File: src/scss/_grid.scss [hunk 5]
```diff
@@ -1,26 +1,95 @@
...
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgrey;
+
+  h1 {
+    text-align: center;
+  }
```
Comment (al-the-x) [inline src/scss/_grid.scss:37]: These `body` and `h1` styles really don't belong in `_grid.scss`... This file should _just_ contain the styles and classes necessary for the CSS grid... as if it was a feature!

**Comment by jeffworley** [comment]
Couldn't get my local to register that there was a merge conflict.  As a result, I am closing this PR, creating a pt2 feature branch in the class repo, and creating a PR there to continue making progress.

**Pull request closed without merging**